source/_posts/2019-10-25-rip-hassbian.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "R.I.P Hassbian"
+description: "The end of Hassbian"
+date: 2019-10-25 01:11:03
```

Comment:
We should make sure this date is in the past when we hit publish, or else it won't be published.

Reply:
Current date should be good as it is but time might need to be changed depending on the timezone of the servers.

Reply:
Time is UTC.